packages/web-config-server/src/tests/apiV1/dataBuilders/generic/count/countEvents.test.js

```diff
@@ -27,10 +27,12 @@ const MOCK_EVENTS = [
 const dataServices = [{ isDataRegional: true }];
 const entity = {};
 const query = { organisationUnitCode: 'PG' };
+const models = {
+  project: { findOne: sinon.stub().callsFake(async () => ({ entity_hierarchy_id: 'xxx' })) },
```

Comment:
No need to change this, but a reminder you can also do

```suggestion
  project: { findOne: sinon.stub().resolves({ entity_hierarchy_id: 'xxx' }) },
```

Reply:
Thanks! I had this thought but didn't bother to look it up 😓